src/main.py

```diff
@@ -11,15 +12,35 @@
 # while developing.
 BROWSABLE = os.environ.get("LOCAL_BROWSE_API")
 
-app = FastAPI()
+# Create a FastAPI instance with metadata
+app = FastAPI(
+    title="IDPD-API-POC",
+    description="A proof of concept API layer to manage content negotiation across IDPD resources and provide an abstraction to 3rd party data services. This API provides access to various metadata, including datasets, editions, publishers, and topics.",
+    version="0.0.1",
+)
 
 
 @app.get("/datasets", response_model=Optional[schemas.Datasets])
-def datasets(
+def get_all_datasets(
```

Comment:
it wasn't you but we've got a rogue additional `response.status_code = status.HTTP_200_OK` in this function, can you take it out as part of this pr please, thanks